Program.cs

```diff
@@ -1,7 +1,32 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿/// <summary>Program that swaps the number from one index position to another</summary>
+class Program {
+   /// <summary>The entry point of the program</summary>
+   static void Main () {
+      int number = GetInput ("Enter a positive number : ");
+      char[] arr = Convert.ToString (number).ToCharArray ();
+      int max = arr.Length - 1;
+      Console.WriteLine ($"Enter the index positions to be swapped within 0 to {max}:");
+      int from = GetInput ("FromPosition = ", max);
+      int to = GetInput ("ToPosition = ", max);
+      Swap (arr, from, to);
+      Console.WriteLine ($"Index swapped number: {new string (arr)}");
+   }
+
+   ///<summary>Gets the input from the user until it is a positive number</summary>
+   ///<param name="prompt">Displays the prompt for the user to enter the input</param>

```

Comment:
```suggestion
   /// <param name="prompt">Displays the prompt for the user to enter the input</param>

```
@Harshini-Jesi Always leave a space after /// or //